Sources/TuistKit/Services/BuildService.swift

```diff
@@ -58,7 +65,10 @@ final class BuildService {
         }
 
         let buildableSchemes = buildGraphInspector.buildableSchemes(graph: graph)
-        logger.log(level: .notice, "Found the following buildable schemes: \(buildableSchemes.map(\.name).joined(separator: ", "))")
+
+        if listSchemes {
```

Comment:
I'd remove the flag entirely and print this with `logger.debug`. The simpler the interface the better. The 2 scenarios that I see with this workflow are:
- I use the right scheme and therefore I don't care about the list of schemes.
- I introduce an invalid scheme and therefore I'd like to know what schemes are available to use the right one.

As a user I'd not expect the schemes to be printed in the first case. I know what I'm doing. In the second case we can print something like:

```
We couldn't find scheme xxx in the following list of found schemes: A, B, C. If your scheme is not in the list, it's possible that the scheme is not part of the project's dependency graph.
```

Reply:
That makes sense, will do!